app/controllers/find_local_council_controller.rb

```diff
@@ -1,7 +1,7 @@
 require "postcode_sanitizer"
 
 class FindLocalCouncilController < ApplicationController
-  before_filter :setup_navigation_helpers
+  before_filter -> { setup_navigation_helpers BASE_PATH }
```

Comment:
The `setup_navigation_helpers` method has been centralised in the `ApplicationController`, therefore it needs to be provided with a base path to work with. This change passes in the base path from this controller to the method in the `ApplicationController` using the `before_filter`.